.idea/libraries/javax_inject.xml

```diff
@@ -1,5 +1,5 @@
 <component name="libraryTable">
-  <library name="javax.inject:javax.inject:1" type="repository">
```

Comment:
Why do you need this? This is a standard way to name a maven-based library.

Reply:
1. For consistency with the overwhelming majority of library names in the project.
2. For readability, since the current name repeats the same word twice.
3. Eliminating the version number from a library names lets you update the library without changing all the orderEntry dependencies in .iml files, which often reach from intellij-community to android, and so on.

Reply:
I would consult @chashnikov first.

Reply:
Sounds good. Thanks!

Reply:
I agree with Dana. I also would add two more reasons:
3a. When you update name of a project library because its version is changed it leads to renaming the library configuration xml file in .idea/libraries and therefore complicates analyzing git history for that library.
3b. When you update name of a project library because its version is changed it may break other projects which are built on top of IntelliJ-community and reuse its project configuration (e.g. IntelliJ IDEA Ultimate project).
So I'll merge this pull request.